### PR TITLE
Adopt to new API

### PR DESCRIPTION
**Pull request by maxim5**
See the details here https://github.com/tensorflow/tensorflow/issues/5742

### CODE REVIEW

**Comment by aymericdamien** [comment]
Hi, Thanks for helping! This way may break for users that doesn't have TensorFlow 0.12. I replaced it with tf.constant_initializer for now.

**Comment by maxim5** [comment]
Sounds good. Thanks!

**Comment by Anjum48** [comment]
Any activity on this? Would love to get batch normalisation running in 0.12...
